webapi/src/test/scala/org/knora/webapi/e2e/admin/PermissionsADME2ESpec.scala

```diff
@@ -15,8 +15,7 @@ import org.knora.webapi.messages.store.triplestoremessages.TriplestoreJsonProtoc
 import org.knora.webapi.sharedtestdata.{SharedOntologyTestDataADM, SharedTestDataADM, SharedTestDataV1}
 import org.knora.webapi.util.AkkaHttpUtils
 import spray.json._
-
-import scala.concurrent.duration._
+import zio._
```

Comment:
is this import really needed here? Doesn't seem to be used

Reply:
It is used. There is a ZIO duration that now needs to be passed to `singleAwaitingRequest`